docusaurus.config.js

```diff
@@ -78,12 +79,12 @@ const config = {
             position: 'left',
             label: 'Tutorial',
```

Comment:
This should be renamed to "Documentation", please also rename the corresponding sidebar, to avoid confusion.
```suggestion
            label: 'Documentation',
```